src/WebJobs.Script/Description/DotNet/DotNetFunctionInvoker.cs

```diff
@@ -170,21 +170,32 @@ private void ResetFunctionValue()
 
         private void RestorePackages()
         {
+            // Kick off the package restore and return.
+            // Any errors will be logged in RestorePackagesAsync
+            RestorePackagesAsync(true)
```

Comment:
Are we only doing this restore on a single instance (the primary instance)? Otherwise we'd have multiple instances all trying at the same time.